docs/en/query_language/alter.md

```diff
@@ -82,144 +88,263 @@ The command is lightweight in a sense that it only changes metadata. To keep the
 rows are ordered by the sorting key expression you cannot add expressions containing existing columns
 to the sorting key (only columns added by the `ADD COLUMN` command in the same `ALTER` query).
 
+### Manipulations With Partitions and Parts {#alter_manipulations-with-partitions}
+
+The following operations are available:
+
+- [DETACH PARTITION](#alter_detach-partition) – Moves a partition to the 'detached' directory and forget it.
+- [DROP PARTITION](#alter_drop-partition) – Deletes a partition.
+- [ATTACH PART|PARTITION](#alter_attach-partition) – Adds a part or partition from the 'detached' directory to the table.
+- [REPLACE PARTITION](#alter_replace-partition) - Copies the data partition from one table to another.
+- [CLEAR COLUMN IN PARTITION](#alter_clear-column-partition) - Resets the value of a specified column in a partition.
+- [FREEZE PARTITION](#alter_freeze-partition) – Creates a backup of a partition.
+- [FETCH PARTITION](#alter_fetch-partition) – Downloads a partition from another server.
+
+A partition is a logic combination of records by a certain criterion. For example, you can set a partition by month, by day or by an hour. Each partition is stored separately in order to simplify manipulations with data.
+
+The partition is specified in the clause `PARTITION BY` when creating a table. For details, see the section [Custom Partitioning Key](../operations/table_engines/custom_partitioning_key.md).
 
-### Manipulations With Partitions and Parts
+A "part" in a table is a part of data in a single partition, sorted by the primary key. When inserting new data to a partition, this data is stored as a separate part (chunk). In 10-15 minutes after inserting, the parts of the same partition are merged into the entire part.
```

Comment:
Definition of "part" using the same word does not look well. At least we could use a synonym like chunk.